api/mapping/models.py

```diff
@@ -255,6 +255,17 @@ class ScanReportConcept(BaseModel):
         blank=True,
     )
 
+    #allow this to be null
```

Comment:
What is the purpose of source_concept?

Reply:
If we want to save a Non-Standard concept along side the Standard-Concept

I've added more code comments about this.

As explained, it was probably beyond the scope of my task as I had thought my task was to just get the full rule generation working again.

At the moment, this isn't used as now I implemented a rejection of Non-Standard concepts when entering concepts. Therefore this will always be null or the same Concept object as the concept